docs/upgrading.md

```diff
@@ -2,6 +2,26 @@
 
 This page lists the breaking API changes between major versions of Kinto.js, as well as upgrade tips.
 
+## 11.x to 12.x
+
+* The `dbPrefix` option in the `Kinto` and `Collection` constructors was dropped in favor of the `dbName` field in the `adapterOptions`.
+
+**Database Schema Change**
+
+This version introduces a major change: instead of having one IndexedDB database per collection (named `{bucket}/{collection}`), we now have only one database (called `KintoDB` by default) which stores records indexed by collection.
```

Comment:
I think it would be valuable to add a little bit more information here about the new database structure. Like Fred Brooks wrote, "Show me your flowcharts and conceal your tables, and I shall continue to be mystified. Show me your tables, and I won’t usually need your flowcharts; they’ll be obvious." My understanding from reading the code is that the database contains two stores, a `records` store and a `timestamp` store. The `records` store contains the records for all collections as plain JS objects with a `_cid` field corresponding to `${bucket}/${collection}`. Is that right? Out of curiosity, why do it this way rather than creating a separate store for `${bucket}/${collection}`? Is it because this way we can enumerate the collections, whereas we cannot enumerate the different stores in an IDB database?

Reply:
> a little bit more information here about the new database structure

Will do!

> Is that right? 

Yes it's right. 

> Out of curiosity, why do it this way rather than creating a separate store for  ${bucket}/${collection}?

That was the approach we had before (https://github.com/Kinto/kinto.js/pull/823#issuecomment-400265375)

Basically, you cannot add new stores without upgrading the version of the database. So if we want to have an arbitrary list of stores (based on the collections that were synchronized) it's a nightmare.